zcash_client_backend/src/data_api/error.rs

```diff
@@ -161,18 +171,25 @@ impl<DE, SE, FE, N> From<InputSelectorError<DE, SE>> for Error<DE, SE, FE, N> {
                 available,
                 required,
             },
+            InputSelectorError::SyncRequired => Error::ScanRequired,
```

Comment:
In this case, it's being raised from the `propose_transaction` function when we can't determine a target or anchor height, because we have no blocks at all.